src/test/ui/consts/const-mut-refs/const_mut_address_of.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0658]: references in constants may only refer to immutable values
+  --> $DIR/const_mut_address_of.rs:24:5
+   |
+LL |     foo().bar();
+   |     ^^^^^ constants require immutable values
+   |
+   = note: see issue #57349 <https://github.com/rust-lang/rust/issues/57349> for more information
+   = help: add `#![feature(const_mut_refs)]` to the crate attributes to enable
```

Comment:
This is wrong now.

Reply:
I was looking for a suitable error in the compiler and couldn't find anything adequate for this.

Reply:
I mean the help message is wrong. The feature gate won't work in constants anymore.

Reply:
Yes, I did some changes locally to show the feature gate error only in constant fns. However we need an error code for the other case.